packages/web-components/fast-components-msft/src/styles/patterns/button.ts

```diff
@@ -51,29 +51,30 @@ export const BaseButtonStyles = css`
         opacity: var(--disabled-opacity);
     }
 
-    .start,
-    .end,
+    ::slotted([slot="start"]),
+    ::slotted([slot="end"]),
```

Comment:
I think we discussed continuing with the class here and pulling the class addition/removal from the template. @EisenbergEffect to confirm though

Reply:
Yes, let's go with the class removal on the container.

Reply:
The class is still present in the template, we're just applying the `width/height/margin-inline` to the slotted content, when it's present.